docs/Containers/Mosquitto.md

```diff
@@ -217,7 +232,7 @@ log_timestamp_format %Y-%m-%dT%H:%M:%S
 
 When `log_dest` is set to 	`stdout`, you inspect Mosquitto's logs like this:
 
-```
+```bash
```

Comment:
Changing this line conflicts with #512. Could you amend the commit not to do this one change?

````diff
diff --cc docs/Containers/Mosquitto.md
index 1c458b3,506a6dc..0000000
--- a/docs/Containers/Mosquitto.md
+++ b/docs/Containers/Mosquitto.md
@@@ -220,13 -233,10 +235,17 @@@ connection_messages fals
  When `log_dest` is set to     `stdout`, you inspect Mosquitto's logs like this:
  
  ```bash
++<<<<<<< HEAD
 +$ cd ~/IOTstack
 +$ docker-compose logs mosquitto
++=======
+ $ docker logs mosquitto
++>>>>>>> pulls
  ```
  
 -Logs written to `stdout` are ephemeral and will disappear when your Mosquitto container is rebuilt, but this type of configuration reduces wear and tear on your SD card.
 +Logs written to `stdout` are stored and persisted to disk as managed by Docker.
 +They are kept over reboots, but are lost when your Mosquitto container is
 +removed or updated.
  
  The alternative, which *may* be more appropriate if you are running on an SSD or HD, is to change `mosquitto.conf` to be like this:
````